pom.xml

```diff
@@ -208,19 +192,16 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>promoted-builds</artifactId>
-      <version>2.27</version>
+      <version>3.2</version>
       <optional>true</optional>
       <exclusions>
         <exclusion>
-          <groupId>org.sonatype.sisu</groupId>
-          <artifactId>sisu-guava</artifactId>
+          <groupId>commons-net</groupId>
+          <artifactId>commons-net</artifactId>
         </exclusion>
         <exclusion>
-          <!-- promoted-builds requests 1.9.2, core requests 1.8.4 -->
-          <!-- use core version, rather than promoted builds version -->
-          <!-- trust this optional reference will be harmless -->
-          <groupId>org.apache.ant</groupId>
-          <artifactId>ant</artifactId>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpclient</artifactId>
```

Comment:
These two exclusions are the main fix. They were transitive dependencies of `promoted-builds`, not a problem for _that_ plugin, but the Maven dependency tree gets confused when also considering `test`-scoped dependency chains.